src/workerd/jsg/value.h

```diff
@@ -477,6 +478,28 @@ class StringWrapper {
     return wrap(context, creator, value.asPtr());
   }
 
+  template <StringLike T>
+  kj::Maybe<T> tryUnwrap(v8::Local<v8::Context> context, const v8::FastOneByteString& handle, T*) {
+    auto& js = Lock::from(context->GetIsolate());
+    auto utf8_length = simdutf::utf8_length_from_latin1(handle.data, handle.length);
+    auto buf = kj::heapArray<char>(utf8_length + 1);
+    buf[utf8_length] = '\0';
+    size_t actual_length =
+        simdutf::convert_latin1_to_utf8_safe(handle.data, handle.length, buf.begin(), buf.size());
+    KJ_DASSERT(actual_length == handle.length);
```

Comment:
This is asserting against the wrong thing isn't it? `handle.length` will be raw bytes while `actual_length` here will be utf8 length?

Reply:
Yes. Fixing it now.